satpy/tests/reader_tests/test_eps_l1b.py

```diff
@@ -163,20 +163,16 @@ def mock_getitem(key):
         mock__init__.return_value = None
         mock__getitem__.side_effect = mock_getitem
         avhrr_reader = satpy.readers.eps_l1b.EPSAVHRRFile()
-        avhrr_reader.sun_azi = None
-        avhrr_reader.sat_azi = None
-        avhrr_reader.sun_zen = None
-        avhrr_reader.sat_zen = None
         avhrr_reader.scanlines = 7
         avhrr_reader.pixels = 2048
 
         with mock.patch.dict("sys.modules", geotiepoints=geotiemock):
             # Get dask arrays
             sun_azi, sun_zen, sat_azi, sat_zen = avhrr_reader.get_full_angles()
             # Convert to numpy array
-            sun_zen_np1 = np.array(avhrr_reader.sun_zen)
```

Comment:
schrödinbug?  How did this ever work in the past?

Reply:
because the sun_zen attribute in avhrr_reader was filled up by the get_full_angles. Now I'm doing this explicitely